src/cassandra.lua

```diff
@@ -74,6 +74,12 @@ local function startup(self)
   return true
 end
 
+local function split(str)
```

Comment:
I think that this function name should be "split_by_port" or it should receive an separator (":")